estuary/utils/general.py

```diff
@@ -399,3 +399,37 @@ def set_story_labels(requested_node_label, results, reverse=False):
         idx += delta
         node_idx += delta
     return results
+
+
+def create_header(item_type, item_id, story_node_story_flow, backward):
+    """
+    Generate a header for the siblings chart.
+
+    :param string item_type: the name of the type of current item
+    :param string item_id: the id of the current item
+    :param dict story_node_story_flow: has forward and backward relationships of current item
+    :param bool backward: determines if the results are in the reverse order of the story flow
```

Comment:
How about `determines if the results are in the reverse order of the story flow` => `determines the relationship direction the passed in node has with the siblings in the story`?